src/test/java/com/scriptbasic/executors/operators/TestOperators.java

```diff
@@ -175,6 +175,9 @@ public void testOperators() throws AnalysisException,
         a("a=1 div 1", 1);
         a("a= 1 div 3", 0);
         a("a=1.0 div 3.0", 0);
+        
+        a("a= \"x\"&\"y\" = \"xy\"", true);
```

Comment:
These test cases does not distinguish the `&` operator from the usual `+` operator. The result is the same for


```java
a("a= \"x\"+\"y\" = \"xy\"", true);
```

Reply:
I added another testcase to demonstrate difference from plus operator.